cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -326,23 +324,30 @@ func (s *Server) hasAccessToNamespace(ctx context.Context, namespace string) (bo
 // isValidChart returns true if the chart model passed defines a value
 // for each required field described at the Helm website:
 // https://helm.sh/docs/topics/charts/#the-chartyaml-file
+// together with required fields for our model.
```

Comment:
This function is checking that we have all the data expected and required to be able to identify the chart. So yes, the helm spec for the fields is great for things like description and name etc., but there are other fields on the model, specific to Kubeapps' model of a chart, such as ID and Repo, which are required to be able to identify/reference the chart.